extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -120,8 +122,9 @@ class AmpBridPlayer extends AMP.BaseElement {
 
     this.feedID_ = user().assert(
         (this.element.getAttribute('data-video') ||
-        this.element.getAttribute('data-playlist')),
-        'Either the data-video or the data-playlist ' +
+        this.element.getAttribute('data-playlist') ||
+        this.element.getAttribute('data-outstream')),
+        'Either the data-video or the data-playlist or the data-outstream ' +
```

Comment:
nit:  `one of the ...` since now there is more than two and `either` won't be right.